<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>385</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1175</width>
    <height>385</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Node name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="node_list"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Snippet title</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Run over(class or bindclass)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="class_cb"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="snippet_list">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Code</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextEdit" name="code">
        <property name="styleSheet">
         <string notr="true">color:rgb(195, 255, 98)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="append_btn">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(79, 238, 116);</string>
        </property>
        <property name="text">
         <string>Append Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="replace_btn">
        <property name="text">
         <string>Replace Code</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>node_list</tabstop>
  <tabstop>class_cb</tabstop>
  <tabstop>snippet_list</tabstop>
  <tabstop>code</tabstop>
  <tabstop>append_btn</tabstop>
  <tabstop>replace_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
